openmdao_book/features/core_features/working_with_components/explicit_component.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>Up to you but <code>optional</code>  might be worth putting in "note" format.</p><p><code></code><code>{Optional}</p><p>Optional message</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I've put it in the note format for now.  We might want to create an `optional` directive if this notation is used more than just here.